server/config.js

```diff
@@ -14,11 +15,11 @@ let config = (function() {
         this.config_table = 'config',
 
         this.mysql = {
-            host: 'localhost',
+            host: 'js2018-group2-mysqldbserver.mysql.database.azure.com',
             port: '3306',
-            user: 'root',
-            password: '',
-            database: 'igraph'
+            user: 'group2@js2018-group2-mysqldbserver',
+            password: 'Mn5Ehoq3BEsKnJV4XQvW',
```

Comment:
I am not sure how we should handle this maybe @syxolk has an idea. We need to connect to this database, otherwise the test would fail.